src/govuk/components/accordion/_index.scss

```diff
@@ -190,77 +252,57 @@
-      color: $govuk-link-hover-colour;
-      text-decoration: underline;
+    // Add toggle link with Chevron icon on left.
+    .govuk-accordion__section-toggle {
+      @include govuk-font($size: 19, $line-height: 1);
+      margin-top: govuk-spacing(1);
+      margin-bottom: govuk-spacing(1);
 
-      // This needs to come after the text-decoration property otherwise
-      // text-decoration, as a shorthand property, resets it to auto
-      @include govuk-link-hover-decoration;
-      text-underline-offset: $govuk-link-underline-offset;
-    }
+      color: $govuk-link-colour;
 
-    // For devices that can't hover such as touch devices,
-    // remove hover state as it can be stuck in that state (iOS).
-    @media (hover: none) {
-      .govuk-accordion__section-button:hover {
-        text-decoration: none;
+      &:active {
+        background: $govuk-link-active-colour;
       }
     }
 
-    .govuk-accordion__controls {
-      text-align: right;
+    // By default span is inline, these properties allow the elements to wrap visually and
+    // allow the width(and focus states) to correctly adjust the the content instead of being full width.
+    .govuk-accordion__section-toggle,
+    .govuk-accordion__section-heading-focus-wrapper,
+    .govuk-accordion__section-summary {
+      float: left;
+      clear: left;
+      text-align: left;
     }
 
-    // Display an icon to the right of each header to indicate open/closed status,
-    // and as an additional affordance.
-    .govuk-accordion__icon {
-      position: absolute;
-      top: 50%;
-      right: 15px;
-      width: 16px;
-      height: 16px;
-      margin-top: -8px;
+    // Change the summary subheading size.govuk-accordion__section-summary
+    .govuk-accordion__section-summary {
+      @include govuk-responsive-margin(1, "top");
+      @include govuk-responsive-margin(2, "bottom");
+      @include govuk-typography-responsive($size: 19, $override-line-height: 1);
     }
 
-    .govuk-accordion__icon:after,
-    .govuk-accordion__icon:before {
-      content: "";
-      box-sizing: border-box;
-      position: absolute;
-      top: 0;
-      right: 0;
-      bottom: 0;
-      left: 0;
-      width: 25%;
-      height: 25%;
-      margin: auto;
-      border: 2px solid transparent;
-      background-color: govuk-colour("black");
+    // Hide body of expanded sections
+    .govuk-accordion__section-content {
+      display: none;
+      @include govuk-responsive-padding(0, "top");
+      @include govuk-responsive-padding(8, "bottom");
     }
 
-    .govuk-accordion__icon:before {
-      width: 100%;
+    // Show the body of expanded sections
+    .govuk-accordion__section--expanded .govuk-accordion__section-content {
+      display: block;
     }
 
-    .govuk-accordion__icon:after {
-      height: 100%;
+    // Remove the bottom margin from the last item inside the content
+    .govuk-accordion__section-content > :last-child {
```

Comment:
We moved this rule here but didn't delete the same rule from line 32, the duplicate should be removed.